test/db/archos/linux-x64/cmd_i

```diff
@@ -7,12 +7,14 @@ e io.cache=true
 "(show_fname fname; wz $0; wtf `prz`; o `prz`; i~file; ij; rm `prz`)"
 .(show_fname A\x1b¢\302\200€𝄞\363\240\201\201\\.bin)
 .(show_fname B\x1b¢\302\200€𝄞\363\240\201\201\\.bin)
-# INVALID FILENAME .(show_fname B\x1b¢\302\200\200€𝄞\363\240\201\201\\.bin)
+.(show_fname B\x1b¢\302\200\201€𝄞\363\240\201\201\\.bin)
```

Comment:
Same here

Reply:
Oh I intend to fix the JSON after this pr, and it's easier to see whether the fix works if the highlighted byte is different from the previous byte ... I'm making the change now just in case I forget later